Tests/FoundationTests/CalendarExtensionTest.swift

```diff
@@ -28,5 +28,34 @@ class CalendarExtensionTests: XCTestCase {
         XCTAssert(calendar.numberOfDaysInMonth(for: febDate) == 28)
         XCTAssert(calendar.numberOfDaysInMonth(for: leapYearDate) == 29)
     }
-	
+    
+    func testNumberOfMonthsInYear() {
+        let gregorian = Calendar(identifier: .gregorian)
+        let hebrew = Calendar(identifier: .hebrew)
+        let ethiopic = Calendar(identifier: .ethiopicAmeteMihret)
+        let today = Date()
+        
+        XCTAssert(gregorian.numberOfMonthsInYear(for: today) == 12)
+        XCTAssert(hebrew.numberOfMonthsInYear(for: today) == 13)
```

Comment:
Should've looked at the tests first :joy:

Reply:
what do you mean?